### DISCUSSION TITLE
Add 'paired' folders in bookmark EXPORT

### DISCUSSION THREAD

**Discussion by pa2eon** [Kovah/LinkAce#675] [Ideas & Feature Requests]
When we make an HTML export.
ALL html bookmarks are placed in 'one line' in the receiving browser.
But most of the saved bookmarks in your browser you can place in 'folders' (see picture )
Is it possible to export to the same situation when we import from LinkAce ?

![afbeelding](https://github.com/Kovah/LinkAce/assets/4719917/d61950ae-ebbb-4cce-b8cf-16626cb8ba37)

**Comment by Kovah** [comment]
Hi, unfortunately, LinkAce has no concept of nested folders that could be used for importing and the export. That might change in the future, but until then it will probably not be added.

**Comment by pa2eon** [comment]
I will check the source code .. to add this option.
The needed info is still in the database (tags and timestamps)  .. so that will give a great export to make 'nested folders'
Yes !! the user have to make the choice first .. nested folder or not.
